### ISSUE TITLE
A quality degradation issue occurred when editing anime characters using Qwen-Image-Edit-2511-Lightning-4steps-bf16.

### ISSUE THREAD

**Issue by yan-yuming** [ModelTC/LightX2V#904]
When using the Qwen-Image-Edit-2511-Lightning-4steps-V1.0-bf16.safetensors model for image editing, it was found that the quality of the edited anime/2D style characters was significantly lower compared to the full-step model.

<img width="1073" height="808" alt="Image" src="https://github.com/user-attachments/assets/e076a4dd-3bc3-4eea-8548-cac16ed4c9bd" />

<img width="1069" height="802" alt="Image" src="https://github.com/user-attachments/assets/96094078-80e6-4d7f-973d-d0c94aca2491" />

<img width="1064" height="805" alt="Image" src="https://github.com/user-attachments/assets/efc53496-d3df-4d25-9050-e673cb7a2528" />

### code 
from typing import Optional, List
from dataclasses import dataclass
import math
import time
import torch
from PIL import Image
from diffusers import QwenImageEditPlusPipeline, FlowMatchEulerDiscreteScheduler

LIGHT_SCHEDULER_CONFIG = {
    "base_image_seq_len": 256,
    "base_shift": math.log(3),
    "invert_sigmas": False,
    "max_image_seq_len": 8192,
    "max_shift": math.log(3),
    "num_train_timesteps": 1000,
    "shift": 1.0,
    "shift_terminal": None,
    "stochastic_sampling": False,
    "time_shift_type": "exponential",
    "use_beta_sigmas": False,
    "use_dynamic_shifting": True,
    "use_exponential_sigmas": False,
    "use_karras_sigmas": False,
}
LIGHT_SCHEDULER = FlowMatchEulerDiscreteScheduler.from_config(LIGHT_SCHEDULER_CONFIG)

@dataclass
class QwenImageEditConfig:
    """QwenImageEdit initialization config."""

    model_path: str = "Qwen/Qwen-Image-Edit-2511"
    lora: Optional[str] = None

class QwenImageEdit:
    """Qwen Image Edit wrapper."""

    def __init__(self, config: QwenImageEditConfig):
        if not torch.cuda.is_available():
            raise RuntimeError("CUDA is not available!")

        self.config = config
        self.device = "cuda"

        self.pipe = QwenImageEditPlusPipeline.from_pretrained(
            config.model_path, torch_dtype=torch.bfloat16
        ).to(self.device)
        self.pipe.set_progress_bar_config(disable=None)

        if config.lora:
            self.pipe.scheduler = LIGHT_SCHEDULER
            self.pipe.load_lora_weights(config.lora)
            self.pipe.fuse_lora()
            self.pipe.unload_lora_weights()

    @torch.inference_mode()
    def infer(
        self,
        images: List[Image.Image],
        prompt: str,
        negative_prompt: str = " ",
        width: int = 1024,
        height: int = 1024,
        guidance_scale: float = 1.0,
        true_cfg_scale: float = 4.0,
        num_inference_steps: int = 4,
        num_images_per_prompt: int = 1,
        seed: int = 2333,
    ) -> Image.Image:
        if not (0 < len(images) <= 3):
            raise ValueError("Number of images must be between 1 and 3.")

        output = self.pipe(
            image=images,
            prompt=prompt,
            negative_prompt=negative_prompt,
            width=width,
            height=height,
            guidance_scale=guidance_scale,
            true_cfg_scale=true_cfg_scale,
            num_inference_steps=num_inference_steps,
            num_images_per_prompt=num_images_per_prompt,
            generator=torch.manual_seed(seed),
        )
        return output.images[0]

These are my generation parameters and code.  diffusers==0.37.0.dev0
"parameters": {
    "width": 896,
    "height": 1344,
    "num_inference_steps": 4,
    "guidance_scale": 1.0,
    "true_cfg_scale": 4.0,
    "seed": 2333,
},

**Comment by pete-finesse** [comment]
how did you solve this? Im running into the same issue. Thought it works nicely on an RTX 6000 PRO, when i try to do the same flow on an H100 it degrades the quality
